disco/state.py

```diff
@@ -285,6 +285,11 @@ def on_guild_member_add(self, event):
         if event.member.guild_id not in self.guilds:
             return
 
+        if (self.guilds[event.member.guild_id].member_count is not UNSET and
```

Comment:
In what situation can a guild be tracked in state but have member_count be unset? I'd think it shouldn't be possible but I don't have a great memory around when Discord sends member_count.

Reply:
As long as it's populated by GuildCreate it should have member_count so i'll remove that.